ember-cli-build.js

```diff
@@ -26,7 +26,8 @@ module.exports = function(defaults) {
     babel: babelOptions,
 
     'ember-cli-babel': {
-      includePolyfill: process.env.EMBER_ENV === 'production'
+      includePolyfill:
+        process.env.EMBER_ENV === 'production' || Boolean(process.env.CI)
```

Comment:
You want me to drop this change?

Reply:
Yeah, that'd be great! CI is using all evergreen browsers now, so the polyfill is probably not necessary these days.